arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts

```diff
@@ -133,6 +134,18 @@
 	vddio-supply = <&pm8953_l6>;
 };*/
 
+&i2c_3 {
+	status = "okay";
+
+	touchscreen@41 {
+		compatible = "ili2117";
```

Comment:
compatible should be `ilitek,ili2117` according to the docs. This is likely (without checking) the i2c compatible which shouldn't be used

Reply:
Done. Thanks!